specification/keyvault/data-plane/readme.md

```diff
@@ -45,6 +45,23 @@ input-file:
   - Microsoft.KeyVault/preview/7.3-preview/securitydomain.json
   - Microsoft.KeyVault/preview/7.3-preview/storage.json
 ```
+
+### Tag: package-7.2
+
+These settings apply only when `--tag=package-7.2` is specified on the command line.
+
+``` yaml $(tag) == 'package-7.2'
+input-file:
+- Microsoft.KeyVault/preview/7.2/certificates.json
```

Comment:
These should reference the "stable" folder.

Reply:
Once thatis done, I should be able to use the OpenAPI Hub Diff'ing tool to make sure things are correct. I suspect you just copied the files and changed the api-version, but want to verify everything to sign off, of course.

Reply:
Thanks